docs/content/getting-started/tuples-api-best-practices.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Managing Tuples and Invoking API Best Practices
+description: Best Practices of Managing Tuples and Invoking APIs
+sidebar_position: 8
+slug: /getting-started/tuples-api-best-practices
+---
+
+import {
+  DocumentationNotice,
+  ProductName,
+  ProductNameFormat,
+  RelatedSection,
+} from "@components/Docs";
+
+# Best Practices of Managing Tuples and Invoking APIs
+
+<DocumentationNotice />
+
+The following list outlines some guidelines and best-practices for using <ProductName format={ProductNameFormat.ShortForm}/>:
+
+- Do not store Personal Identifiable Information in tuples
+- Always specify authorization model ID whenever possible
+- Avoid using intersection or exclusion in ListObjects authorization model
+
+## Do Not Store Personal Identifiable Information in Tuples
+
+You can use any string for user and object identifiers, but we strongly recommend that you do not store Personal Identifiable Information (PII) tuples for the following reasons:
```

Comment:
Can you add a disclaimer before or after this section that the docs does use readable identifiers to make it easier to follow?

Reply:
done